### PR TITLE
Added article about nft social volume

### PR DESCRIPTION
**Pull request by ilya-l-san**
## Checklist

- [ ] My article available in Navigation sidebar and in root article of selected category ([how to do it in README](https://github.com/santiment/academy#add-an-article-into-navigation-sidebar))
- [ ]	My article have [metadata](https://github.com/santiment/academy#metadata) (title, description, date when updated and author)
- [ ]	All added/updated links in article are exist, images shows correctly

### CODE REVIEW

File: src/docs/metrics/nft-social-volume/index.md [hunk 1]
```diff
@@ -0,0 +1,97 @@
+---
+title: NFT Social Volume
+author: Ilya
+date: 2022-05-15
+description: The amount of messages/documents containing a mention of NFT Collection
+# REF metrics-hub/metricshub/nft_collections_social_volume.py
```
Comment (Partysun) [inline src/docs/metrics/nft-social-volume/index.md:6]: @ilya-lysko pls remove this. #line

File: src/docs/metrics/nft-social-volume/index.md [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+
+## Access
+
+[Restricted Access](/metrics/details/access#restricted-access).
+
+---
+
+## Measuring Unit
+
+Amount of documents that mention the given NFT Collection.
```
Comment (Partysun) [inline src/docs/metrics/nft-social-volume/index.md:39]: Btw do we use the same words in Social Volume Measuring Unit section? @ilya-l-san

File: src/docs/metrics/nft-social-volume/index.md (hunk 1 above)
Comment (ilya-l-san) [reply src/docs/metrics/nft-social-volume/index.md]: done

File: src/docs/metrics/nft-social-volume/index.md (hunk 2 above)
Comment (ilya-l-san) [reply src/docs/metrics/nft-social-volume/index.md:38]: Social Volume Measuring Unit is very close: `"Amount of documents that mention the given text pattern."`

**Pull request merged**